src/legacy/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -768,6 +768,13 @@ function discoverController(
     $scope.time = timefilter.getTime();
   };
 
+  $scope.toMoment = function (datetime) {
+    const format = config.get('dateFormat');
+    if (moment.isMoment(datetime)) return datetime.format(format);
```

Comment:
The `moment` constructor allows also passing a `moment` instance, so this can be simplified to:

```js
return moment(datetime).format(config.get('dateFormat'));
```